Jenkinsfile

```diff
@@ -467,25 +468,50 @@ def doAndroidBuildInDocker(String abi, String buildType, boolean runTestsInEmula
 }
 
 def doBuildWindows(String buildType, boolean isUWP, String platform, boolean runTests) {
-    def cmakeDefinitions;
     def warningFilters = [];
-    if (isUWP) {
-      cmakeDefinitions = '-DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION=10.0'
-      // warning on benchmark binaries that we don't care about
-      warningFilters = [excludeMessage('Publisher name .* does not match signing certificate subject')]
```

Comment:
This doesn't have an equivalent on the rhs. Did the warning get fixed?

Reply:
We don't build the test programs on UWP now.  However I will reinsert the filter, so it is not forgottten.